go/chat/storage/storage_ephemeral_purge.go

```diff
@@ -58,11 +82,6 @@ func (s *Storage) EphemeralPurge(ctx context.Context, convID chat1.ConversationI
 	if err != nil {
 		return nil, err
 	}
-	// End of the line
-	if newPurgeInfo.MinUnexplodedID == maxMsgID {
```

Comment:
we don't want to delete here since we'll lose our place in the chat and have to sweep over the entire chat on the next ephemeral message that explodes